File: CS007/project2/P2A3_KELLY_3938535.java
/* Author: John Kelly Jr
Filename: P2A3_KELLY_3938535
Description: Create a simulations of a craps game.
This particular program will allow the user to try his or her luck. */

import java.util.Scanner; // Import scanner
import java.util.Random; // Import random number generator

public class P2A3_KELLY_3938535
{
	public static void main(String[] args)
	{
		Scanner input = new Scanner(System.in);

		System.out.println("Hello, please enter your first name:");
		String firstName = input.next();

		System.out.println("Hello, please enter your last name:");
		String lastName = input.next();

		System.out.println("Hello " + firstName + " " + lastName +
					", welcome to John Kelly "
				+ "Jr's craps simulation. Best of luck!");

		System.out.println("Here are some instructions:");
		System.out.println("If you roll a 7 or 11 on the first roll you win.");
		System.out.println("If you roll a 2, 3, or 12 on the first roll you lose.");
		System.out.println("If you roll any other number, the \"come out roll\""
					+ " becomes the \"point number.\"");
		System.out.println("You must keep rolling until you roll a 7, "
				+ "in which you lose, or roll the \"point number\""
				+ ", in which you win.");

		// Welcome message


		Random R = new Random();

		int dice = R.nextInt(11) + 2; // Two die


		System.out.println("You rolled a: " + dice);


		if(dice == 7 || dice == 11) // A win
			System.out.println("Congratulations you won!");


		else
		{
			if (dice == 2 || dice == 3 || dice == 12) // A loss
				System.out.println("I'm sorry, you lost.");

			else
				System.out.println("Your new point number is: " + dice
				+ ". Good luck!");

		}
	}
}

File: CS008/P2/P2A3_KELLY_3938535.py
#P2
#John Kelly Jr
#The program will allow the user to test their luck in a game of Rockymon.
#Lecture: Mon/Wed 4:30-5:45
#Lab: Wed 2-2:50

from random import randint

print('\t\t\t     Rockymon')
print('\t\t\tBy: John Kelly Jr')
name=input('\nPlease enter in your name: ')
print('\nHello,',name,'!')
print('\nRules of the game:')
print('   You will first roll two die. If you roll a 5 or 10 on your first try you win!')
print('   If you roll a 2, 4, or 11 on your first try you lose.')
print('   Otherwise, the number you roll becomes the \"match number\".')
print('   You must keep rolling until you roll the \"match numer\", which means you win,')
print('   or a 5, which means you lose.\n')
print('\t\t\t     Good luck!')
#Welcome message


dice = randint(2,12)#random number generator 
print('You rolled a:', dice)

#Game (1 roll)
if dice == 5 or dice == 10:
    print('Congrats, you win!')
elif dice==2 or dice==4 or dice==11:
    print('I\'m sorry, but you lose.')
else:
    print('Your new match number is:', dice)
